packages/template/src/components/user-button.tsx

```diff
@@ -69,29 +69,18 @@ function UserButtonInnerInner(props: UserButtonProps & { user: CurrentUser | nul
 
   return (
     <DropdownMenu>
-      <DropdownMenuTrigger className="outline-none stack-scope">
+      <DropdownMenuTrigger className="flex-grow w-12 outline-none stack-scope rounded-lg hover:bg-muted/50 transition-colors p-1 -m-1">
         <div className="flex gap-2 items-center">
           <UserAvatar user={user} />
           {user && props.showUserInfo &&
-            <div className="flex flex-col justify-center text-left">
+            <div className="flex-grow w-0 flex flex-col justify-center text-left">
               <div className="max-w-40 truncate">{user.displayName}</div>
               <div className="max-w-40 truncate text-sm text-gray-500">{user.primaryEmail}</div>
             </div>
           }
         </div>
       </DropdownMenuTrigger>
-      <DropdownMenuContent className="stack-scope">
-        <DropdownMenuLabel>
-          <div className="flex gap-2 items-center">
-            <UserAvatar user={user} />
-            <div>
-              {user && <Typography className="max-w-40 truncate">{user.displayName}</Typography>}
-              {user && <Typography className="max-w-40 truncate" variant="secondary" type='label'>{user.primaryEmail}</Typography>}
-              {!user && <Typography>{t('Not signed in')}</Typography>}
-            </div>
-          </div>
-        </DropdownMenuLabel>
-        <DropdownMenuSeparator />
+      <DropdownMenuContent className="stack-scope w-60" align="end">
```

Comment:
don't do backwards-incompatible changes on the components in the packages as they will break our users who depend on them